docs/components/card.md

```diff
@@ -1308,6 +1308,9 @@ Shows implementations of the publication element.
         <span itemprop="name">Lorem ipsum dolor sit amet</span>
       </div>
       <div class="meta-item meta-published">
```

Comment:
"meta-published" describes the publication time of a card item, instead of reusing it a new e.g. "meta-distance" should be introduced.